lib/pluginmanager/install.rb

```diff
@@ -85,8 +86,9 @@ def install_gems_list!(install_list)
     gemfile.save
 
     bundler_options = {:install => true}
-    bundler_options[:without] = [] if development?
+    bundler_options[:without] = [] if development? || local?
```

Comment:
no reason, leftover from research during the initial faces of this. Thanks a lot for noticing.